frontend/pages/_app.tsx

```diff
@@ -47,7 +48,7 @@ export default function App({
   const { locale, defaultLocale, pathname } = router;
   const messages = locales[locale];
   return (
-    <ChakraProvider>
+    <ChakraProvider theme={theme}>
```

Comment:
Question: Can we add the theme prop to the `.storybook/preview.tsx` file? It would help keep our styles consistent between storybook and the app!